db/schema.rb

```diff
@@ -80,17 +73,20 @@
     t.integer "sport_facilities"
     t.float "working_time"
     t.float "students_per_teacher"
+    t.string "powiat", null: false
+    t.string "gmina", null: false
+    t.string "miejscowosc", null: false
     t.index ["institution_type_id"], name: "index_institutions_on_institution_type_id"
   end
 
   create_table "users", force: :cascade do |t|
     t.string "email", default: "", null: false
     t.string "encrypted_password", default: "", null: false
     t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "reset_password_sent_at", precision: nil
+    t.datetime "remember_created_at", precision: nil
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
That's strange. I wonder where are those changes coming from?

Reply:
This must have come from some new cop inside rubocop. I didn't introduce those changes manually and I didn't use any external commands that might have modified the schema this way. This seems to affect all datetime fields except created_at and updated_at in all tables.

Reply:
The same cop has removed specified precision from all created_at and updated_at fields in all tables.

Reply:
We an disable rubocop for schema

Reply:
Now that you mention that I noticed that we already did. does 
excluding 
    - 'db/**/*'
not exclude files in db directory but only files in subdirectiories in the db directory? Should we specifically exclude the schema file?